R/as_html.R

```diff
@@ -80,15 +80,16 @@ as_html <- function(x,
 
   nlh <- mf_nlheader(mat)
   nc <- ncol(x) + 1
+  nr <- length(mf_lgrouping(mat))
 
   # Structure is a list of lists with rows (one for each line grouping) and cols as dimensions
-  cells <- matrix(rep(list(list()), (nlh + nrow(x)) * (nc)), ncol = nc)
+  cells <- matrix(rep(list(list()), (nr) * (nc)), ncol = nc)
```

Comment:
Don't really need the extra brackets
```suggestion
  cells <- matrix(rep(list(list()), nr * nc), ncol = nc)
```